packages/my-joyent/scripts/patch-webpack-config.js

```diff
@@ -0,0 +1,120 @@
+const webpack = require('webpack');
```

Comment:
We should consider detaching this into it's own package.

Reply:
Issue created : https://github.com/yldio/joyent-portal/issues/640